index.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="styles.css">
+        <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-0evHe/X+R7YkIZDRvuzKMRqM+OrBnVFBL6DOitfPri4tjfHxaWutUpFmBp4vmVor"
+      crossorigin="anonymous"
+    />
+        <title>Fisrt doc</title>
+    </head>
+    <body>
+        <header id="header">
+            <nav class="navbar navbar-expand navbar-light bg-white fixed-top" id="nav-bar">
```

Comment:
I think the nav tag should be replaced by the header tag for proper following of semantic HTML rules